earth_enterprise/src/server/wsgi/wms/ogc/wms.cfg

```diff
@@ -0,0 +1,9 @@
+# Toggle to enable and disable certificate validation
+VALIDATE_CERTIFICATE False
```

Comment:
Should this default to True?

Reply:
No. By default it should just work out of the box. Configuring certificate validation is optional.